pkg/nsx/headerconfig.go

```diff
@@ -37,6 +38,9 @@ func (headerConfig *HeaderConfig) Process(req *http.Request) error {
 	if headerConfig.nsxEnablePartialPatch {
 		req.Header.Set("nsx-enable-partial-patch", "true")
 	}
+	if headerConfig.hiddenApi {
+		req.Header.Set("hidder-api", "true")
```

Comment:
hidden?

Reply:
removed, it's used to verify that SDK client could add extra header